{
    "docs": "https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/pylint.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test pylint` [[explain](https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/pylint.html)] failed with 2 errors:", 
            "output": "tests/unit/modules/conftest.py:6:0: unused-import: Unused import os\ntests/unit/modules/test_get_servers.py:11:0: unused-import: Unused module_mock imported from commun_test"
        }
    ], 
    "verified": false
}
